Jenkinsfile

```diff
@@ -1,4 +1,5 @@
 buildPlugin(configurations: [
-    [platform: 'linux', jdk: '11'],
-    [platform: 'windows', jdk: '11'],
-])
\ No newline at end of file
+    [platform: 'linux', jdk: 21],
+    [platform: 'linux', jdk: 11],
+    [platform: 'windows', jdk: 17],
```

Comment:
What's the rationale behind this? Why 17 on Windows (and not 11, for example)?

Reply:
Thanks for asking.  There are several reasons why this change chooses to test with Java 21 and Java 17 without testing Java 11 on ci.jenkins.io.

* Java 11 byte code is generated by the Java 17 and Java 21 compilers due to the configuration of the plugin parent pom.   Java 11 byte code is thus being tested with Java 17 and Java 21 runtimes
* Java 11 compilation and test is performed for this plugin at least once a week in the plugin compatibility tests that are part of the Jenkins plugin bill of materials.  The 200+ plugins included in the plugin bill of materials are acting as "safety checks" in case there is some surprise due to not compiling with Java 11 on ci.jenkins.io .  Many of the plugins in the plugin BOM enabled this pattern 4-6 weeks ago.  We've seen no failures due to the removal of Java 11 compilation and test on ci.jenkins.io
* It is important to keep infrastructure costs as low as we can for ci.jenkins.io .  Testing only one Windows configuration and one Linux configuration is a way of reducing those infrastructure costs.  Choosing Java 17 and Java 21 improves the JDK coverage and assures that we're "looking forward"

Reply:
> test with Java 21 and Java 17 without testing Java 11 on ci.jenkins.io

Then you forgot to remove the Java 11 configuration entry there in the `Jenkinsfile`?

Reply:
> Then you forgot to remove the Java 11 configuration entry there in the `Jenkinsfile`?

Thanks for catching that.  Fixed in a26f4c133563597ab8a1bc6239bda6601e19c6a6